src/main/kotlin/xyz/crossward/util/KotlinUtils.kt

```diff
@@ -1,6 +1,10 @@
 package xyz.crossward.util
 
+import java.time.ZoneId
+import java.time.ZonedDateTime
 import java.util.*
 
 // Convert java Optional into kotlin null safety
-fun <T> Optional<T>.unwrap(): T? = orElse(null)
\ No newline at end of file
+fun <T> Optional<T>.unwrap(): T? = orElse(null)
+
+fun currentDateInEST(): ZonedDateTime = ZonedDateTime.now(ZoneId.of("US/Eastern"))
```

Comment:
It looks like [ZoneID](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html) for EST does not include Daylight Savings and will always be UTC-5 regardless of if it's currently daylight savings. 

Maybe we need to just make our own function that [checks the date](https://en.m.wikipedia.org/wiki/Eastern_Time_Zone) and does this:
If not daylight savings: New York Time is UTC-5 (EST)
If daylight savings: New York Time is UTC - 4 (EDT)

Reply:
Actually this might already work, I was looking at documentation for "EST" not "US/Eastern" which [seems to account for DST](https://stackoverflow.com/questions/10545960/how-to-tackle-daylight-savings-using-timezone-in-java)